### DISCUSSION TITLE
Feature request: updates sensor

### DISCUSSION THREAD

**Discussion by ishioni** [dougiteixeira/proxmoxve#112] [Feature request] [answered]
Hi

Thanks for this, i see you're trying to upstream this, and hopefully that'll happen soon. In the meanwhile, would it be possible to add a sensor that counts the number of packages to update?

**Comment by dougiteixeira** [comment]
At the moment I don't have the time to invest in these improvements, but I'm happy to merge them if you want to contribute.

**Comment by dougiteixeira** [comment, marked as answer]
I have included a binary sensor that will be active when updates are available and a sensor with the total number of updates and package list as attribute (https://github.com/dougiteixeira/proxmoxve/pull/158) and will be available in the next release.
